rebar.config

```diff
@@ -15,3 +15,7 @@
         {dev_mode, false},
         {include_src, false}
        ]}.
+
+{plugins, [
+  {gradualizer, {git, "https://github.com/aleklisi/Gradualizer", {branch, "master"}}}
```

Comment:
Also, I'd definitely go for the original `josefs` repo instead of your fork, I don't know how much trouble you might have at keeping your up-to-date. They just pushed a fix for that OTP22 problem you were having :slightly_smiling_face: